dist/react-filterbar.js

```diff
@@ -11135,9 +11146,40 @@ var FilterBarStore = exports.FilterBarStore = (function () {
     updateFilter: {
       value: function updateFilter(filterUid, propKey, propValue) {
         this.filters[filterUid][propKey] = propValue;
+        if (propKey === "value") this.deactivateQuickFiltersBasedOnFilterValue(filterUid, propValue, this.activeQuickFilters());
         this.emitChange();
       }
     },
+    deactivateQuickFiltersBasedOnFilterValue: {
+      value: function deactivateQuickFiltersBasedOnFilterValue(filterName, filterValue, quickFilters) {
+        quickFilters.map(function (quickFilter) {
+          Object.keys(quickFilter.filters).map(function (quickFilterName) {
+            var quickFilterFilter = quickFilter.filters[quickFilterName];
+            if (quickFilterFilter.filter === filterName) {
+              if (typeof quickFilterFilter.value === "object") {
```

Comment:
Also it would be good if this could be refactored .. into a few more functions possilbly.
Its a bit large and complex to grasp easily